index.php

```diff
@@ -1,17 +1,6 @@
-<?php require_once __DIR__ . '/config/variables.php'; ?>
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <?php require_once __DIR__ . '/template/head.phtml'; ?>
-</head>
-<body>
-<?php require_once __DIR__ . '/template/menu.phtml'; ?>
-<main id="home">
-    <?php
-    $content = 'home';
-    require_once __DIR__ . '/template/page.phtml';
-    ?>
-</main>
-<?php require_once __DIR__ . '/template/footer.phtml'; ?>
-</body>
-</html>
+<?php
+require_once __DIR__ . '/config/variables.php';
+$content = isset($_GET['page']) ? $_GET['page'] : 'index';

```

Comment:
@ReginaBorisova 
this is correct.   in the page.phtml  require one file with the name that is the value of the `$content` variable.